internal/cli/get.go

```diff
@@ -120,10 +124,23 @@ func (o *GetOptions) Validate(args []string) error {
 		if kind != DeviceKind && kind != FleetKind {
 			return fmt.Errorf("owner can only be specified when fetching devices and fleets")
 		}
-		if (kind == DeviceKind || kind == FleetKind) && len(name) > 0 {
```

Comment:
There is no need to check the kind; it's already verified on the previous if condition.